src/App.jsx

```diff
@@ -47,11 +52,17 @@ export function App() {
 		});
 	}, [listToken]);
 
+	// const handleCreateList = () => {
+	// 	const newToken = generateToken();
+	// 	setListToken(newToken);
+	// 	navigate('/list');
+	// };
+
```

Comment:
We forgot to remove this commented code, this will be removed in the next commit on this branch.